custom_components/ocpp/translations/i-default.json

```diff
@@ -60,5 +61,25 @@
         "abort": {
             "single_instance_allowed": "Only a single instance is allowed."
         }
+    },
+    "exceptions": {
+        "invalid_ocpp_key": {
+            "message": "Invalid OCPP key"
+        },
+        "ocpp_call_error": {
+            "message": "OCPP call failed: {message}"
+        },
+        "get_variables_error": {
+            "message": "Failed to get variable: {message}"
+        },
+        "set_variables_error": {
+            "message": "Failed to set variable: {message}"
+        },
+        "profile_not_supported": {
+            "message": "Setting custom charging profiles not supported"
+        },
+        "dual_profile_not_supported": {
```

Comment:
I think I can get rid of both profile_not_supported and this. The former can probably just be implemented by converting OCPP 1.6 profile to 2.0 profile, and the latter cannot even happen because nothing calls set_charge_rate with both limit_amps and limit_watts set.